src/backend/legendary/games.ts

```diff
@@ -634,11 +635,17 @@ class LegendaryGame extends Game {
           ? wineExec.replaceAll("'", '')
           : wineExec
 
-      wineFlag.push(
-        ...(wineType === 'proton'
-          ? ['--no-wine', '--wrapper', `'${wineBin}' run`]
-          : ['--wine', wineBin])
-      )
+      if (wineType === 'proton') {
+        wineFlag.push(...['--no-wine', '--wrapper', `'${wineBin}' run`])
```

Comment:
We can remove these `...[___]` syntax now that we don't have the ternary operator:

```suggestion
        wineFlag.push('--no-wine', '--wrapper', `'${wineBin}' run`)
```